controllers/dockermachine_controller.go

```diff
@@ -59,12 +61,36 @@ func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 // SetupWithManager will add watches for this controller
 func (r *DockerMachineReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&infrastructurev1alpha1.DockerMachine{}).
+		For(&infrastructurev1alpha2.DockerMachine{}).
 		Watches(
-			&source.Kind{Type: &infrastructurev1alpha1.DockerMachine{}},
+			&source.Kind{Type: &infrastructurev1alpha2.DockerMachine{}},
 			&handler.EnqueueRequestsFromMapFunc{
-				ToRequests: util.MachineToInfrastructureMapFunc(capiv1alpha2.SchemeGroupVersion.WithKind("Machine")),
+				ToRequests: MachineToDockerMachine(capiv1alpha2.SchemeGroupVersion.WithKind("Machine")),
 			},
 		).
 		Complete(r)
 }
+
+// MachineToDockerMachine watches v1alpha2 machines and enqueues associated DockerMachines
+func MachineToDockerMachine(gvk schema.GroupVersionKind) handler.ToRequestsFunc {
```

Comment:
This should be removed with the reversion to `util.MachineToDockerMachine()` above.